lib/model/QubitObject.php

```diff
@@ -150,23 +150,18 @@ public function save($connection = null)
 
   protected function insert($connection = null)
   {
-    if (!isset($connection))
-    {
-      $connection = QubitTransactionFilter::getConnection(QubitObject::DATABASE_NAME);
-    }
-
```

Comment:
Why removing this one entirely? I'd the same change that is done below.

Reply:
`$connection` is only used in the local scope as an argument to `BaseObject::insert()` and self::insertSlug()`, and both methods already use the same `$connection = Propel::getConnection()` pattern, so there's it's unnecessary here.